test/api/cpp/smt2_compliance.cpp

```diff
@@ -55,15 +52,15 @@ void testGetInfo(cvc5::Solver* solver, const char* s)
 {
   std::unique_ptr<SymbolManager> symman(new SymbolManager(solver));
 
-  std::unique_ptr<Parser> p(ParserBuilder(solver, symman.get(), true).build());
-  p->setInput(Input::newStringInput(
-      "LANG_SMTLIB_V2_6", string("(get-info ") + s + ")", "<internal>"));
-  assert(p != NULL);
-  std::unique_ptr<Command> c = p->nextCommand();
-  assert(c != NULL);
-  cout << c.get() << endl;
-  stringstream ss;
+  InputParser p(solver, symman.get());
+  std::stringstream ssi;
+  ssi << "(get-info " << s << ")";
+  p.setStreamInput("LANG_SMTLIB_V2_6", ssi, "<internal>");
+  std::unique_ptr<Command> c = p.nextCommand();
+  assert(c != nullptr);
+  std::cout << c.get() << std::endl;
+  std::stringstream ss;
   c->invoke(solver, symman.get(), ss);
-  assert(p->nextCommand() == NULL);
-  cout << ss.str() << endl << endl;
+  assert(p.nextCommand() == nullptr);
+  std::cout << ss.str() << std::endl << std::endl;
```

Comment:
Ditto. Wouldn't it be better to assert that the string is what it is expected to be?

Reply:
I'm not sure its worthwhile to test for an expected output here. This is just calling various `get-info` commands, whose (default) values are prone to change, and some are unsupported.  I guess the original intention of the test was just to ensure we don't give an exception.